source/server/admin/admin.cc

```diff
@@ -113,6 +113,7 @@ AdminImpl::AdminImpl(const std::string& profile_path, Server::Instance& server,
           server_.api().randomGenerator())),
       profile_path_(profile_path), stats_(Http::ConnectionManagerImpl::generateStats(
                                        "http.admin.", *server_.stats().rootScope())),
+      null_overload_manager_(NullOverloadManager(server.threadLocal(), false)),
```

Comment:
No need to call the ctor directly. e.g. `NullOverloadManager(...)`.

You should be able to do `null_overload_manager_(...)`

Reply:
Changed